lib/server.rb

```diff
@@ -6,6 +6,12 @@ class Server < Grape::API
   format :json
   prefix :api
 
+  helpers do
+    def authenticate
+      error!("Unauthorized", 401) unless headers["X-Api-Token"] == ENV["API_KEY"] || ENV["API_KEY"].nil? && !headers["X-Api-Token"].nil?
```

Comment:
I've set this up so that if no api_key is supplied in the env vars, the api is essentially open - is this the way we want to approach this? Should we do the opposite - ie if no api_key is in the env vars, the api is locked down completely? Am I overthinking this since it's unlikely there will be a scenario where the env var is empty?

Reply:
As discussed during this morning tech weekly. We should lock the API down by default and if we need to make it public (i.e in the future we might want to develop and api for organisations to use to import data directly to their own backend services, we would develop that later)

Reply:
Just changed this so that if no api_key is supplied in the env vars the api is locked down